types/react-icon-base/index.d.ts

```diff
@@ -1,15 +1,20 @@
 // Type definitions for react-icon-base 2.0
 // Project: https://github.com/gorangajic/react-icon-base#readme
-// Definitions by: Alexandre Paré <https://github.com/apare>
+// Definitions by: Alexandre Paré <https://github.com/apare>, Karol Janyst <https://github.com/LKay>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.1
 
-import * as React from 'react';
+import * as React from "react";
 
-export interface IconBaseProps {
-    color?: string;
-    size?: string | number;
-    style?: React.CSSProperties;
+export = IconBaseClass;
```

Comment:
It used to be but compiled and released version of the library doesn't export anything as default. At least in the current version. Default export becomes undefined which causes runtime errors. Tested in actual app code.